wifiphisher/pywifiphisher.py

```diff
@@ -420,6 +430,15 @@ def start(self):
         try:
             if self.opmode.internet_sharing_enabled():
                 self.network_manager.internet_access_enable = True
+                # Set up an automatic MITM attack if `-m/--mitminterface` is present. 
+                #
+                # We are already handling the chosen interface as an internetInterface.
+                # Here we are also protecting the rest of the detected interfaces.
+                #  (i.e. prevent NetworkManager from managing them)
+                if args.mitminterface:
+                    restoftheinterfaces=interfaces.NetworkManager._name_to_object.items()
+                    for interface in restoftheinterfaces.remove(args.mitminterface):
```

Comment:
Are you using Python3? On Python3  `.items()`  returns “views” instead of lists, hence this will throw an error `dict_items object has no attribute 'remove'`. More here: https://docs.python.org/3/whatsnew/3.0.html#views-and-iterators-instead-of-lists

Reply:
I apologise, I might have used an old version to test this by mistake.
I refactored this block to avoid creating the unnecessary list as well.